modules/apps/adaptive-media/adaptive-media-blogs-web/src/main/java/com/liferay/adaptive/media/blogs/web/internal/counter/BlogsAMImageCounter.java

```diff
@@ -53,16 +58,28 @@ public int countExpectedAMImageEntries(long companyId) {
 
 		Property mimeTypeProperty = PropertyFactoryUtil.forName("mimeType");
 
-		dynamicQuery.add(
-			mimeTypeProperty.in(
-				_amImageMimeTypeProvider.getSupportedMimeTypes()));
+		Set<String> supportedMimeTypes = new HashSet<>(
```

Comment:
Can you replace this chunk of code (and its duplicate down below) by something like:
```
dynamicQuery.add(
        mimeTypeProperty.in(
                ArrayUtil.filter(
                        _amImageMimeTypeProvider.getSupportedMimeTypes(),
                        _amImageValidator::isProcessingSupported));
```